spk/tvheadend/Makefile

```diff
@@ -2,9 +2,9 @@ SPK_NAME = tvheadend
 SPK_SHORT_VERS = 4.3
 SPK_GIT_HASH = 9ed76c0
 SPK_GIT_DATE = 20201011
-SPK_VERS = $(SPK_SHORT_VERS).$(SPK_GIT_DATE)~$(SPK_GIT_HASH)
+SPK_VERS = $(SPK_SHORT_VERS).$(SPK_GIT_DATE)
```

Comment:
Revert this for previous behavior

Reply:
I can revert everything but this. dsm7 requires digits not letters or symbols. I think I copied the error message in the git commit message:

> Failed to load pkg info, version=[4.3.20201011~9ed76c0-26], correct-format=[^\d+(\.\d+){0,5}(-\d+)?$]

We still have revision numbers and the date

Reply:
I guess the hash could be discarded entirely so only the date is being kept.
Or add a bit of fun with converting hex to decimal using `echo $((16#9ed76c0))`

Still, the git hash needs to be kept for  the help web page in order to provide actual version from:
`TVH_VERS = $(SPK_SHORT_VERS)~$(SPK_GIT_HASH)`